go/vt/sqlparser/parse_test.go

```diff
@@ -226,11 +228,11 @@ var (
 			input: "select -1 from t where b = -2",
 		}, {
 			input:                      "select - -1 from t",
-			output:                     "select - -1 from t",
+			output:                     "select 1 from t",
```

Comment:
This was a load-bearing quirk: mysql echoes back the literal expression in the result schema, and this test is attempting to make sure we captured it correctly.

If you want to change these tests to not do that then fine, but you need additional tests to make sure we're capturing the select expression in that case